## Task 6: Bayesian Estimates
$\xi_{17}=19$

$\xi_{18}=69$

$\xi_{19}=73.46$

The task involves estimating the scale parameter $\theta$ of a Gamma distributed variable $X$ using a Bayesian approach, where the prior for $\theta$ is also specified.

Parameters

•	Data Distribution (Likelihood): $X \sim \Gamma(\alpha_L=3, \beta_L=1/\theta)$. 

o	$\alpha_L$ (Shape parameter) $= 3$.

•	Prior Distribution: $\theta \sim \Gamma(\alpha_{pr}, \beta_{pr})$. 

o	$\alpha_{pr} = \xi_{17} = 19$.

o	$\beta_{pr} = \xi_{18} = 69$.

$\rightarrow$ $\theta \sim \Gamma(\alpha=\xi_{17}=19, \beta=\xi_{18}=69)$. 

•	Observed Sample Data: $N = 10$ observations, with observed mean $\bar{x} = \xi_{19} = 73.46$.

•	Sample Sum: $\sum x_i = N \cdot \bar{x} = 10 \times 73.46 = 734.6$.


In [1]:
#import libraries
import numpy as np

In [2]:
# Input Parameters
N = 10
x_bar = 73.46
sum_x = N * x_bar   # 734.6
alpha_L = 3         # Shape parameter of the data distribution (Gamma)

# Prior Parameters (Interpreted as Inverse Gamma Conjugate Parameters)
alpha_pr = 19
beta_pr = 69

**Step 1:**

The Posterior Distribution $\theta$:

In Bayesian statistics, the posterior distribution $P(\theta | \mathbf{x})$ is proportional to the Likelihood $\mathcal{L}(\theta | \mathbf{x})$ multiplied by the Prior $P(\theta)$ <cite data-cite="con:coursebook"> (Advanced Statistics DLMDSAS01, 2025, p. 97)</cite>.
$$ P(\theta | \mathbf{x}) \propto \mathcal{L}(\theta | \mathbf{x}) \cdot P(\theta) $$

Conjugate Prior Assumption:

The random variable $X$ follows a Gamma distribution $\Gamma(k, \theta)$ parameterized by the scale parameter $\theta$ <cite data-cite="con:coursebook"> (Advanced Statistics DLMDSAS01, 2025, p. 26)</cite>. For this parameterization (estimating the scale parameter $\theta$), the mathematically consistent conjugate prior is the Inverse Gamma distribution <cite data-cite="con:coursebook"> (Advanced Statistics DLMDSAS01, 2025, p. 110)</cite>.

Since the prompt explicitly specifies a Gamma prior with parameters $\xi_{17}$ and $\xi_{18}$, we proceed by mapping these parameters to the required Inverse Gamma form: $IG(\alpha_{IG}, \beta_{IG})$.

A. Likelihood Function $\mathcal{L}(\theta | \mathbf{x})$

The PDF for a single Gamma observation $x_i$ with shape $\alpha_L=3$ and scale $\theta$ is: $$ f(x_i | \theta) = \frac{1}{\Gamma(3) \theta^3} x_i^{3-1} e^{-x_i/\theta} $$
The joint likelihood for $N=10$ i.i.d. observations is: $$ \mathcal{L}(\theta | \mathbf{x}) = \prod_{i=1}^{N} f(x_i | \theta) \propto \theta^{-3N} e^{-\frac{1}{\theta} \sum x_i} $$ Substituting $N=10$ and $\sum x_i = 734.6$: $$ \mathcal{L}(\theta | \mathbf{x}) \propto \mathbf{\theta^{-30} e^{-\frac{734.6}{\theta}}} $$

B. Prior Distribution $P(\theta)$

We use the form of the Inverse Gamma distribution $IG(\alpha_{pr}, \beta_{pr})$ that is conjugate to the likelihood: $$ P(\theta) \propto \theta^{-(\alpha_{pr}+1)} e^{-\frac{\beta_{pr}}{\theta}} $$ Substituting $\alpha_{pr}=19$ and $\beta_{pr}=69$: $$ P(\theta) \propto \theta^{-(19+1)} e^{-\frac{69}{\theta}} = \mathbf{\theta^{-20} e^{-\frac{69}{\theta}}} $$

C. Posterior Distribution $P(\theta | \mathbf{x})$

We multiply the core likelihood and core prior components: $$ P(\theta | \mathbf{x}) \propto \left( \theta^{-30} e^{-\frac{734.6}{\theta}} \right) \left( \theta^{-20} e^{-\frac{69}{\theta}} \right) $$ Combining the exponents of $\theta$ and the constants in the exponential term: $$ P(\theta | \mathbf{x}) \propto \theta^{-(30+20)} e^{-\frac{734.6 + 69}{\theta}} $$ $$ P(\theta | \mathbf{x}) \propto \theta^{-50} e^{-\frac{803.6}{\theta}} $$
This result is the kernel of an Inverse Gamma distribution $IG(\alpha_{post}, \beta_{post})$, where the exponents must match the form $\theta^{-(\alpha_{post}+1)} e^{-\beta_{post}/\theta}$.

Calculation of Posterior Parameters:

1.	Shape term: $-(\alpha_{post} + 1) = -50 \implies \alpha_{post} = 49$.

2.	Scale term: $\beta_{post} = 803.6$.
$$ \mathbf{\text{Posterior Distribution: } \theta \sim IG(\alpha_{post}=49, \beta_{post}=803.6)} $$


**Step 2:**

Bayes Point Estimate (Square-Error Loss Function):

The Bayes point estimate associated with the square-error loss function is the mean (or expected value) of the posterior distribution <cite data-cite="con:coursebook"> (Advanced Statistics DLMDSAS01, 2025, p. 97)</cite>.

The expected value $E[\theta]$ for an Inverse Gamma distribution $IG(\alpha, \beta)$ is: $$ E[\theta] = \frac{\beta}{\alpha - 1} \quad \text{for } \alpha > 1 $$

Mathematical Calculation: $$ E[\theta] = \frac{\beta_{post}}{\alpha_{post} - 1} = \frac{803.6}{49 - 1} = \frac{803.6}{48} \approx \mathbf{16.7417} $$


In [3]:
# Posterior Parameters
alpha_post = alpha_L * N + alpha_pr
beta_post = sum_x + beta_pr
print(f"\nPosterior Inverse Gamma Shape (alpha_post): {alpha_post}")
print(f"Posterior Inverse Gamma Scale (beta_post): {beta_post}")

# Bayes Estimate based on Square-Error Loss (Mean of Posterior)
E_theta = beta_post / (alpha_post - 1)

print(f"\n--- Bayes Point Estimate (Square-Error Loss) ---")
print(f"E[θ] = {E_theta:.4f}")



Posterior Inverse Gamma Shape (alpha_post): 49
Posterior Inverse Gamma Scale (beta_post): 803.5999999999999

--- Bayes Point Estimate (Square-Error Loss) ---
E[θ] = 16.7417


**Step 3:**

Bayes Point Estimate (Mode of Posterior Distribution):

The Bayes point estimate using the mode is the value of $\theta$ that maximizes the posterior density function <cite data-cite="con:coursebook"> (Advanced Statistics DLMDSAS01, 2025, p. 54)</cite>.

The mode $\text{Mode}[\theta]$ for an Inverse Gamma distribution $IG(\alpha, \beta)$ is: $$ \text{Mode}[\theta] = \frac{\beta}{\alpha + 1} \quad \text{for } \alpha > 0 $$

Mathematical Calculation: $$ \text{Mode}[\theta] = \frac{\beta_{post}}{\alpha_{post} + 1} = \frac{803.6}{49 + 1} = \frac{803.6}{50} = \mathbf{16.072} $$


In [4]:
# Bayes Estimate based on Mode of Posterior
Mode_theta = beta_post / (alpha_post + 1)

print(f"\n--- Bayes Point Estimate (Mode) ---")
print(f"Mode[θ] = {Mode_theta:.4f}")



--- Bayes Point Estimate (Mode) ---
Mode[θ] = 16.0720


**Summary of Results**

| Query Part	| Calculation	| Result
| --- | --- | --- |
| a) Posterior Distribution	| $IG(\alpha_{post}, \beta_{post})$	| $IG(\alpha=49, \beta=803.6)$ |
| b) Bayes Estimate (Square-Error Loss)	| Mean: $\beta / (\alpha - 1)$	| 16.7417 |
| c) Bayes Estimate (Mode)	| Mode: $\beta / (\alpha + 1)$	| 16.0720 |

**Justification of Tools:** 

Python's NumPy library was used for basic numerical computation. The derivations rely on the principles of Bayes' Rule and the property of conjugate priors <cite data-cite="con:coursebook"> (Advanced Statistics DLMDSAS01, 2025, p. 109)</cite>.

Since the problem involves estimating the scale parameter ($\theta$) of a Gamma distribution, the analytical solution requires the conjugate prior to be an Inverse Gamma distribution, whose functional form defines the structure of the resulting posterior. The formulas for the mean and mode of the Inverse Gamma distribution determine the final Bayes point estimates.

